src/components/chips/js/chipsController.js

```diff
@@ -397,6 +397,7 @@ MdChipsCtrl.prototype.hasMaxChipsReached = function() {
  */
 MdChipsCtrl.prototype.validateModel = function() {
   this.ngModelCtrl.$setValidity('md-max-chips', !this.hasMaxChipsReached());
+  this.ngModelCtrl.$validate(); // rerun any registered validators
```

Comment:
It might be a good idea to check if the `$validate` method exists here. If I remember correctly, it was introduced in Angular 1.4.